### ISSUE TITLE
dynamic in upload url

### ISSUE THREAD

**Issue by purzyk** [flyntwp/flynt#217]
Eveytime i upload img or any other asset and i try to use it in page components ive got 404 (Not Found) - because flynt Theme is adding word dynamic into url
for example
https://test.local/wp-content/uploads/dynamic/2020/01/P1040406_DxO-1440x0-c-default.jpg

![image](https://user-images.githubusercontent.com/33529405/72053440-538f4380-32c7-11ea-8615-4036b9149096.png)

**Comment by domtra** [comment]
@badger612 i had a deeper look into the issue. if you check out the dynamicResize branch and flush the permalinks, nginx will serve the images correctly with the nginx config, that is suggested by wordpress.org. However, there are some nginx configs that still lead to errors. you can identify, if it is a config issue by opening the image in the browser. if it is a 404 from nginx, it is a nginx config issue. in this case, a location rule like the following should help:
```
location ~ "^(.*)/wp-content/uploads/(.*)$" {
        try_files $uri $uri/ /index.php$is_args$args;
}
```

Please let me know if this fixes your issues. Then we should be able to merge the dynamicResize branch into master.

**Comment by Jensderond** [comment]
@domtra this works for me! **Important step is to flush the permalinks!**

**Comment by domtra** [comment]
closed by #214

**Comment by spsaucier-bakkt** [comment]
Is there a way to get this to work with WPEngine, where you don't have access to nginx config?
